package.json

```diff
@@ -60,6 +60,7 @@
   },
   "devDependencies": {
     "@chakra-ui/cli": "2.1.4",
+    "@chakra-ui/styled-system": "2.2.5",
```

Comment:
This is a phantom dependency issue in Chakra itself. [I created an issue in their repo](https://github.com/chakra-ui/chakra-ui/issues/6427), but this is the workaround while it gets fixed.